components/forms/Form/Form.tsx

```diff
@@ -5,8 +5,15 @@ import { validateOnSubmit, getErrorList, setFocusOnErrorMessage } from "../../..
 import { submitToAPI } from "../../../lib/integration/helpers";
 import { Button, Alert } from "../index";
 import { logMessage } from "../../../lib/logger";
-import { FormValues, InnerFormProps, DynamicFormProps, Responses } from "../../../lib/types";
+import {
+  FormValues,
+  InnerFormProps,
+  DynamicFormProps,
+  Responses,
+  ReCaptchaResponse,
+} from "../../../lib/types";
```

Comment:
There was a change in the code a while ago that allows us to use `@lib/` for imports, instead of some pretty deep relative paths. It would be great to update these on files as we make changes to them.